Gems/Terrain/Code/Source/TerrainRenderer/TerrainClipmapManager.cpp

```diff
@@ -327,26 +461,38 @@ namespace Terrain
         m_clipmaps[ClipmapName::MacroColor] =
             AZ::RPI::AttachmentImage::Create(*pool.get(), imageDesc, macroColorClipmapName, nullptr, nullptr);
 
-        imageDesc.m_arraySize = aznumeric_cast<uint16_t>(m_detailClipmapStackSize);
-
-        AZ::Name detailColorClipmapName = AZ::Name("DetailColorClipmaps");
-        m_clipmaps[ClipmapName::DetailColor] =
-            AZ::RPI::AttachmentImage::Create(*pool.get(), imageDesc, detailColorClipmapName, nullptr, nullptr);
-
         imageDesc.m_format = AZ::RHI::Format::R16G16_SNORM;
         imageDesc.m_arraySize = aznumeric_cast<uint16_t>(m_macroClipmapStackSize);
 
         AZ::Name macroNormalClipmapName = AZ::Name("MacroNormalClipmaps");
         m_clipmaps[ClipmapName::MacroNormal] =
             AZ::RPI::AttachmentImage::Create(*pool.get(), imageDesc, macroNormalClipmapName, nullptr, nullptr);
+    }
+
+    void TerrainClipmapManager::InitializeDetailClipmapImages()
+    {
+        AZ::Data::Instance<AZ::RPI::AttachmentImagePool> pool = AZ::RPI::ImageSystemInterface::Get()->GetSystemAttachmentPool();
+
+        AZ::RHI::ImageDescriptor imageDesc;
+        imageDesc.m_bindFlags = AZ::RHI::ImageBindFlags::ShaderReadWrite;
+        imageDesc.m_size = AZ::RHI::Size(m_config.m_clipmapSize, m_config.m_clipmapSize, 1);
 
+        // TODO: Test and find the most suitable precision for color map.
```

Comment:
Nit: Shouldn't check in "TODO" comments. The work should either get done or the comment should be rewritten as more of a future suggestion. Ex:
```
// The format for the color map is chosen to be R8G8B8A8_UNORM for now, but it may
// need to be changed to something else at some point if X is discovered to be true.
```

Reply:
I forgot to remove this one. I'll update.

Reply:
Still here...

Reply:
I removed the other one. Because the function is split into 2 to allow separate update for detail and macro.